.github/pull_request_template.md

```diff
@@ -0,0 +1,15 @@
+<!--
+Thank you for your pull request.
+Please see http://cldr.unicode.org/index/process for general
+information on contributing to CLDR.
+
+You will be automatically asked to sign the contributors license before the PR is accepted.
+- sign: https://cla-assistant.io/unicode-org/cldr
+- license: http://www.unicode.org/copyright.html#License
+-->
+
+##### Checklist
+
+- [ ] Issue filed: https://unicode-org.atlassian.net/browse/CLDR-_____

```

Comment:
These days the link can be just: `[CLDR-1234]` and the bot will linkify it. But, I think this is fine.

Frequent contributors may want to replace the line with: `- [X] Issue Filed: [CLDR-1234]`  but I don't think we want to keep the template concise.

Reply:
Thanks Steven. I didn't know if the bot was hooked up to CLDR yet.
I suppose it could be changed later if needed.
Thanks!